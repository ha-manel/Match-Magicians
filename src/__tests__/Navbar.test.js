import React from 'react';
import renderer from 'react-test-renderer';
// import { render, fireEvent, screen } from '@testing-library/react';
import Navbar from '../components/navbar';
// import App from '../App';
import '@testing-library/jest-dom';

describe('Test of navbar', () => {
  it('test if the navbar is rendered', () => {
    const navbar = renderer
      .create(<Navbar />)
      .toJSON();
    expect(navbar).toMatchSnapshot();
  });

  // it('test the home link', () => {
  //   const { container } = render(<App />);

  //   fireEvent.click(screen.getByText('Home'));

  //   const home = container.querySelector('#home-page');
  //   expect(home).toBeInTheDocument();
  // });
});
