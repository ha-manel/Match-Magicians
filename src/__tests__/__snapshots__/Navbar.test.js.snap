// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of navbar test if the navbar is rendered 1`] = `
<header>
  <h1
    id="logo"
  >
    Math Magicians
  </h1>
  <nav>
    <a
      className="nav-link"
      href="#/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      className="nav-link"
      href="#/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      className="nav-link"
      href="#/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </nav>
</header>
`;
